packages/bot-engine/src/utils/index.js

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
In my opinion this can become confusing, we're going to end up importing everything from a single file if I'm not mistaken?

Reply:
@aaron-binary  my plan is to export 
   `services, utils, ... ` in separate object and import it like this :
`import { utils } from 'deriv-bot-engine`` 
and the use it as `utils.something`, but i will do it in next PR